test/e2e_flags.go

```diff
@@ -30,18 +31,19 @@ import (
 )
 
 const (
-	// E2ETestNamespacePrefix is the namespace prefix used for running all e2e tests.
-	E2ETestNamespacePrefix = "e2e-ns"
 	// DefaultClusterChannelProvisioner is the default ClusterChannelProvisioner we will run tests against.
-	DefaultClusterChannelProvisioner = "in-memory-channel"
-	// DefaultBrokerName is the name of the Broker that is automatically created after the current namespace is labeled.
-	DefaultBrokerName = "default"
+	DefaultClusterChannelProvisioner = InMemoryChannelProvisioner
 )
 
 var logger = logging.FromContext(context.Background()).Named("eventing-e2e-testing")
 
 // validProvisioners is a list of provisioners that Eventing currently support.
-var validProvisioners = []string{DefaultClusterChannelProvisioner}
+var validProvisioners = []string{
+	InMemoryChannelProvisioner,
```

Comment:
`in-memory` should be in this list too. In fact, it is the default provisioner.

Reply:
Done.
Also moved `validProvisioners` to `constants.go` to avoid maintaining in too many places.